File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/uncomment/direct/Java/C/s826491904_commented.c
#include <stdio.h>

int main() {
    int N, max, min, num;
    scanf("%d", &N);
    scanf("%d", &max);
    min = max;
    for (int i = 1; i < N; i++) {
        scanf("%d", &num);
        if (num > max) max = num;
        if (num < min) min = num;
    }
    printf("%d\n", max - min);
    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/uncomment/direct/Java/Go/s826491904_commented.go
package main

import (
	"fmt"
)

func main() {
	var N int
	fmt.Scan(&N)
	var max, min int
	fmt.Scan(&max)
	min = max
	for i := 1; i < N; i++ {
		var num int
		fmt.Scan(&num)
		if num > max {
			max = num
		}
		if num < min {
			min = num
		}
	}
	fmt.Println(max - min)
}

// End of Code
